src/ziggurat/config.clj

```diff
@@ -142,7 +146,9 @@
    :origin-topic
    :poll-timeout-ms-config
    :producer
-   :thread-count])
+   :thread-count
+   :enabled
```

Comment:
can :enabled conflict with some other enabled key in the future? Should we name it as ssl-enabled, just to avoid confusion?